Header header
uint8[] pdo_in_raw
uint8[] pdo_out_raw
